onnxruntime/test/testdata/custom_execution_provider_library/symbols.txt

```diff
@@ -0,0 +1 @@
+OrtSessionOptionsAppendExecutionProvider_MyEP
```

Comment:
Is this needed given that your EP is going to be dynamically loaded and you're using the GetProvider interface to create and register it?

Reply:
right, we can remove it.